helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -149,6 +149,33 @@ def _read_group_stage_ids(self, stages, domain, order):
         string="Automatic last update", default=datetime.now()
     )
 
+    ticket_change = fields.Datetime(
+        string="Next stage date", compute="_compute_next_stage"
+    )
+
+    @api.depends("stage_id")
```

Comment:
@cristina-domatix i think it can be done while depending [only on the needed attributes](https://github.com/odoo/odoo/blob/13.0/odoo/api.py#L184) (`auto_next_number`, `auto_next_type`, `auto_next_stage_id`) and not the full `stage_id` object as to avoid unnecesary calculations